docker-bench-security.sh

```diff
@@ -19,6 +19,7 @@ dir_name=`dirname ${this_path}`    ## Dir where this file is
 myname=`basename ${this_path}`     ## file name of this script.
 logger="${myname}.log"
 
+export PATH=/bin:/sbin:/usr/bin
```

Comment:
This line makes this not work on my mac. The reason is that `command -v docker` now files since my `docker` binary lives in `/usr/local/bin/docker`

```✗ command -v docker
/usr/local/bin/docker
```

Why do we need this change?